### DISCUSSION TITLE
Where can I find the decode/descriptions for qualityControl values for wind speed?

### DISCUSSION THREAD

**Discussion by gpalovchik** [weather-gov/api#274] [Q&A]
**NOTE**
Please do not report operational issues (such as outages) here. We can not support these issues via this forum. Please report these as described here: https://weather-gov.github.io/api/reporting-issues

**Please describe what you're trying to do.**
A clear and concise description of what your issue is. Ex. I'm looking for forecast data...

**Comment by jlawler2** [comment]
I just ran into this myself. Data returned:
"dewpoint": {
            "value": 7,
            "unitCode": "unit:degC",
            "qualityControl": "qc:V"
        },

qc:V has no definition, and is therefore useless unless we know what V means vs Z, etc. I also noticed that regardless of whether the boolean flag is set to true, false, or left off completely, it always returns the qualityControl node. ;)

**Comment by StephenClouse** [comment]
The codes correspond to the [MADIS](https://madis.ncep.noaa.gov/) quality control codes as fully described at [this link](https://madis.ncep.noaa.gov/madis_sfc_qc_notes.shtml).

Code summary:

```  
  No QC available:

   Z - Preliminary, no QC

  Automated QC checks:

   C - Coarse pass, passed level 1
   S - Screened, passed levels 1 and 2
   V - Verified, passed levels 1, 2, and 3
   X - Rejected/erroneous, failed level 1
   Q - Questioned, passed level 1, failed 2 or 3

       where level 1 = validity
             level 2 = internal consistency, temporal consistency,
                       statistical spatial consistency checks
             level 3 = spatial consistency check

Subjective intervention:

   G - Subjective good
   B - Subjective bad

  Interpolated/Corrected observations:

   T - Virtual temperature could not be calculated, air temperature passing all QC
       checks has been returned```

**Comment by jlawler2** [comment]
Perfect, thank you!! :D
